v2/emailpassword/common-customizations/sessions/session-verification-in-api/get-session.mdx

```diff
@@ -190,9 +190,24 @@ SuperTokens.init(backendConfig());
 
 export function POST(request: NextRequest) {
     return withPreParsedRequestResponse(request, async (baseRequest: PreParsedRequest, baseResponse: CollectingResponse) => {
-        const session = await Session.getSession(baseRequest, baseResponse);
-        let userId = session.getUserId();
-        return NextResponse.json({});
+        try {
+            const session = await Session.getSession(baseRequest, baseResponse);
+            let userId = session.getUserId();
+            return NextResponse.json({});
+        } catch (err) {
+            try {
+                await errorHandler()(err, baseRequest, baseResponse, (errorHandlerError: Error) => {
+                    if (errorHandlerError) throw err;
+                });
+                return new NextResponse(baseResponse.body, {
+                    status: baseResponse.statusCode,
+                    headers: baseResponse.headers,
+                });
+            } catch (error) {
+                // Handle the error
+                return NextResponse.json({ message: "Somethign went wrong!"}, { status: 500 });
+            }
+        }
```

Comment:
this should go inside the impl of withPreParsedRequestResponse

Reply:
done